server/src/main/java/org/elasticsearch/action/search/FetchLookupFieldsPhase.java

```diff
@@ -87,7 +87,7 @@ private void doRun(List<Cluster> clusters) {
                 : "lookup across clusters only if [ccs_minimize_roundtrips] is disabled";
             for (LookupField lookupField : cluster.lookupFields) {
                 final SearchRequest searchRequest = lookupField.toSearchRequest(clusterAlias);
-                searchRequest.setCcsMinimizeRoundtrips(false);
+                searchRequest.setCcsMinimizeRoundtrips(context.getRequest().isCcsMinimizeRoundtrips());
```

Comment:
What about the assertion above?  Shouldn't it fail if `context.getRequest().isCcsMinimizeRoundtrips()` is true?

Reply:
That assertion should not fail as the cluster alias is empty.

Reply:
Ah right sorry, I confused with the cluster alias that would be set on the lookup index name.